packagesDev/next-config/src/withGraphCommerce.ts

```diff
@@ -161,6 +164,11 @@ export function withGraphCommerce(nextConfig: NextConfig, cwd: string): NextConf
         ],
       }
 
+      config.watchOptions = {
+        ...(config.watchOptions ?? {}),
+        ignored: ['**/.git/**', `**/node_modules/!(@graphcommerce)**`, '**/.next/**'],
+      }
```

Comment:
I think this should be correct code, can we try this out? https://github.com/graphcommerce-org/graphcommerce/blob/canary/packagesDev/next-config/src/withGraphCommerce.ts#L157-L162

Reply:
Yep, this works 👌